src/main/scala/higherkindness/compendium/Main.scala

```diff
@@ -47,7 +47,10 @@ object CompendiumStreamApp {
 
   def stream[F[_]: ConcurrentEffect: Timer: ContextShift]: Stream[F, ExitCode] =
     for {
-      conf                           <- Stream.eval(ConfigSource.default.at("compendium").loadF[F, CompendiumServerConfig])
+      blocker <- Stream.resource(Blocker[F])
```

Comment:
Under the hood, this uses a cached thread pool which seems fitting to read a configuration:

https://github.com/typelevel/cats-effect/blob/master/core/jvm/src/main/scala/cats/effect/internals/BlockerPlatform.scala#L28-L38